wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
-<strong>Note:</strong> Automatically enabled nearing Christmas week (around 17th onwards). Cannot be disabled during this period.</p></td>
-</tr>
-<tr class="even">
-<td><p>Parallax</p></td>
-<td><p>The screen <em>moves</em> with your osu!cursor. Does not work during gameplay.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Menu tips</p></td>
-<td><p>Show a strip of interesting tidbits every time you visit the Main menu. Located at slightly lower part of the giant vibrating osu! logo.</p></td>
-</tr>
-<tr class="even">
-<td><p>Interface voices</p></td>
-<td><p>&quot;Welcome to osu!&quot;/&quot;See you next time.&quot; voice when you open/close osu! respectively.<br />
-VA: ???</p></td>
-</tr>
-<tr class="odd">
-<td><p>osu! music theme</p></td>
-<td><p>Play &quot;<a href="http://soundcloud.com/nekodex/circles">nekodex - circles!</a>&quot; when opening osu! rather than random songs. It is <strong>not a beatmap</strong> and will <strong>not</strong> repeat. Once completed, you can only replay it by re-opening osu!. <a href="http://nekodex.bandcamp.com/track/circles">You can support the artist by bandcamp here</a>.</p>
-<hr>
-<p><strong>[Stable(Fallback) only]</strong><br />
-Play &quot;<a href="http://soundcloud.com/nekodex/welcome-to-osu">nekodex - welcome to osu!</a>&quot; when opening osu! rather than random songs. It is <strong>not a beatmap</strong> and will <strong>always in repeat</strong> until you change the song. Once changed, you can only replay it by re-opening osu!.</p>
-<hr>
-<p><strong>[Nearing Christmas week (December) only]</strong><br />
-Play &quot;<a href="http://soundcloud.com/nekodex/welcome-to-christmas">nekodex - welcome to christmas!</a>&quot; when opening osu! rather than random songs. It is <strong>not a beatmap</strong> and will <strong>not</strong> repeat. Once completed, you can only replay it by re-opening osu!. Cannot be disabled during this period.</p></td>
-</tr>
-</tbody>
-</table>
+| Name             | Effect                                                                                                       |
+|------------------|--------------------------------------------------------------------------------------------------------------|
+| Snow             | Show "snow" effect on the main menu (enabled by default during the holiday season).                          |
+| Parallax         | Show a slight parallax while navigating in-game menus (not during gameplay).                                 |
+| Menu tips        | Show a tip every time you visit the main menu.                                                               |
+| Interface voices | osu! will greet you when you start and quit.                                                                 |
+| osu! music theme | Upon opening osu!, the main theme song will play while on the menu. Entering song select will override this. |
```

Comment:
Entering song select or changing the song or when the song finishes will end the music theme.
Probably should note that once the song changes, you cannot play it again until you reopen osu!